app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -124,7 +124,8 @@ def parse_physical_server(node)
             :firmwares => [] # Filled in later conditionally on what's available
           }
         },
-        :asset_details          => parse_asset_details(node)
+        :asset_details          => parse_asset_details(node),
+        :location_led_state    	=> find_loc_led_state(node.leds)
```

Comment:
This change was merged into https://github.com/ManageIQ/manageiq-providers-lenovo/pull/48 yesterday.